stage2/00-sys-tweaks/01-run.sh

```diff
@@ -23,8 +23,6 @@ else
 	systemctl disable ssh
 fi
 systemctl enable regenerate_ssh_host_keys
-
-sed -i -e "s/\/home\/umbrel/\/home\/${FIRST_USER_NAME}/g" /etc/rc.local
```

Comment:
Leftover from #79 - When I opened that PR, #72 wasn't merged yet, so it used the rc.local for the startup then.